docs/modules/ROOT/pages/how-to-guides/managing-applications/proc-managing_applications.adoc

```diff
@@ -0,0 +1,91 @@
+= Managing applications in secure development environment
+
+This feature enhances the handling of the sensitive content within the development environment. You can efficiently address security fixes while maintaining confidentiality and facilitating the recreation of applications.
+
+Key capabilities of this feature include:
+
+* **Clone affected application:** Cloning affected applications into a secure workspace, enabling developers to tackle security issues without exposing sensitive information.
+
+* **Recreate application with limited visibility:** Recreating applications while minimizing the awareness of their existence, ensuring a secure environment for working on security fixes.
+
+* **Release secure fixes on components:** Release secure fixes on specific components within existing applications, simplifying the process of addressing vulnerabilities without affecting the entire application.
+
+== Cloning applications to a dedicated or all workspaces
+You can clone applications to a dedicated workspace to address security vulnerabilities, test new features, and enhance the application without disrupting the production environment. 
+
+.**Prerequisite**
+
+* You have the maintainer role for the application you are working with.
+* You have access to App Studio.
+* You have downloaded RHTAP link:https://github.com/redhat-appstudio/rhtap-cli/releases[pre-built CLI].
+
+
+.**Procedure**
+
+. Log in to the original workspace where the application was previously being developed using the `oc` CLI tool.
+. To clone the application from one workspace to another, run the following command:
+
++
+[source,bash]
+----
+./rhtap export application build --from <source_tenant_workspace> --to <target_tenant_workspace>
```

Comment:
Is "build" here the example name of their application? Should it be a variable in brackets like the workspaces?

Reply:
Yes. it's the same name of the application :) We should probably be using something that's also not a verb. @gtrivedi88